core/ui/i18n/locales/it.ts

```
@@ -10,8 +10,13 @@ export const it = {
   onboarding_description_4:
     'Vultisig è un portafoglio senza frasi seed. Utilizza invece frammenti facili da gestire. Ogni dispositivo ha il proprio frammento unico che necessita di un backup separato.',
   moreInfo: 'Maggiori informazioni',
+  incorrect_passcode: 'Codice errato',
+  passcodes_do_not_match: 'I codici non corrispondono',
   search_field_placeholder: 'Cerca..',
```

Biome (1.9.4)

[error] This property value named passcodes_do_not_match is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named passcodes_do_not_match


(lint/suspicious/noDuplicateObjectKeys)